packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -412,4 +415,445 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("err opcode", function () {
```

Comment:
we can just use `Err` class instead of `err opcode`. Parent describe mentions - TEAL Opcodes.